prov/net/src/xnet_progress.c

```diff
@@ -950,17 +950,22 @@ static void xnet_progress_cqe(struct xnet_progress *progress,
 			else {
 				bsock->async_prefetch = false;
 				xnet_ep_disable(ep, 0, NULL, 0);
+				return;
 			}
 		} else if (cqe->res <= 0 && !OFI_SOCK_TRY_SND_RCV_AGAIN(-cqe->res)) {
 			if (ep->cur_rx.entry)
 				xnet_complete_rx(ep, cqe->res);
-			else
+			else {
```

Comment:
nit: The coding standard is to add braces to both the if and else statements if either needs them.

Maybe we should use a goto disable in this call to jump to a common point.

Reply:
Thanks. I updated the code to use a goto, as you suggested.